FoodBookApp/UI/Components/SpotCard.swift

```diff
@@ -52,6 +53,8 @@ struct SpotCard: View {
         minTime: 25,
         maxTime: 30,
         distance: 0.5,
-        categories: ["Vegan", "Sandwich", "Bowl", "Healthy", "..."]
+        categories: ["Vegan", "Sandwich", "Bowl", "Healthy", "..."],
```

Comment:
Don't think this is part of this issue, but don't forget we agreed to only show three categories!

Reply:
You are absolutely right!